pubspec.lock

```diff
@@ -373,4 +373,4 @@ packages:
     source: hosted
     version: "3.1.0"
 sdks:
-  dart: ">=2.12.0 <3.0.0"
+  dart: ">=2.14.0 <3.0.0"
```

Comment:
Same thing here. No real reason to up the version number. Its better to stay lower to increase compatibility

Reply:
indeed, I will change this one too.